### ISSUE TITLE
Hub entry contains spurious characters due to truncation

### ISSUE THREAD

**Issue by blinkdog** [ScorpioStation/ScorpioStation#164]
**Issue Description**:
Looks like our status text is just a little too long, and gets truncated in our hub entry.

**When did the problem start happening**:
After #161 

**Extra information**:
![hub-entry](https://user-images.githubusercontent.com/1283521/89607678-b5516500-d838-11ea-9bfc-d6e69596c2a0.png)

I'd bet the `<\` after Green is a closing tag getting clipped by the hub's status text limit.

**Comment by ZomgPonies** [comment]
Changed the blurb from "Join us and" to "Come". Since the blurb is in the config file it doesn't require another PR, and we'll see if it's enough to stop the truncating
